deps.edn

```diff
@@ -67,19 +67,19 @@
                         :extra-deps
                         {clj.native-image/clj.native-image
                          {:git/url "https://github.com/taylorwood/clj.native-image.git"
-                          :sha     "7708e7fd4572459c81f6a6b8e44c96f41cdd92d4"}}}
+                          :sha     "4604ae76855e09cdabc0a2ecc5a7de2cc5b775d6"}}}
 
            ;; Checks
 
            :benchmark {:extra-paths ["benchmark/src"]
                        :extra-deps {clj-http/clj-http {:mvn/version "3.12.3"}
-                                    org.clojure/tools.cli {:mvn/version "1.0.206"}
+                                    org.clojure/tools.cli {:mvn/version "1.0.214"}
                                     incanter/incanter-core {:mvn/version "1.9.3"}
                                     incanter/incanter-charts {:mvn/version "1.9.3"}}
                        :main-opts ["-m" "benchmark.cli"]}
 
-           :format {:extra-deps {cljfmt/cljfmt {:mvn/version "0.8.0"}}
+           :format {:extra-deps {cljfmt/cljfmt {:mvn/version "0.9.2"}}
                     :main-opts ["-m" "cljfmt.main" "check"]}
 
-           :outdated {:extra-deps {com.github.liquidz/antq {:mvn/version "2.2.983"}}
-                      :main-opts ["-m" "antq.core"]}}}
+           :outdated {:extra-deps {com.github.liquidz/antq {:mvn/version "2.2.992"}}
+                      :main-opts ["-m" "antq.core" "--exclude=com.github.liquidz/antq@2.2.999"]}}}
```

Comment:
We could also hold the excluded packages in the `config.edn.` I only put it here for now because it seemed simpler. 

Can also get attached to the packages themselves, see [here](https://github.com/liquidz/antq/blob/main/doc/exclusions.adoc)